studies/templates/emails/notify_admins_of_study_action.html

```diff
@@ -13,10 +13,10 @@
   {% elif action == 'paused' %}
     {{ researcher_name }} has paused the study {{ study_name }}. {{base_url}}{% url 'exp:study-detail' study_id %}
   {% elif action == 'deactivated' %}
-    {{ researcher_name }} has deactivated the study {{ study_name }}. {{base_url}}{{% url 'exp:study-detail' pk=study_id %}
+    {{ researcher_name }} has deactivated the study {{ study_name }}. {{base_url}}{% url 'exp:study-detail' pk=study_id %}
   {% elif action == 'deployed' %}
-     {{ study_name }} has been deployed. {{base_url}}{{% url 'exp:study-detail' pk=study_id %}
+     {{ study_name }} has been deployed. {{base_url}}{% url 'exp:study-detail' pk=study_id %}
```

Comment:
For both this and previewed I linked to the study detail page. Do you think we should link to the page where the study can be taken too? As of right now, at least for preview, this isn't exposed anywhere. I changed your button that took you to the preview to actually trigger the preview job, now there's no way to get to take the preview without URL hacking.

Reply:
Yes, i'll add this.